### PR TITLE
ported the carp suit & ian onesie from ss13

### PR DESCRIPTION
**Pull request by QTNeen**
## About the PR

This PR adds two new suits, one is an Ian onesie that looks ridiculous and the other is a carp hardsuit that slightly increases speed, but is also more vulnerable to attacks.. In memory of old space carp.
Currently there is no way to obtain them outside of being spawned or added to salvage or via mapping: low impact.

**Media**
image

- [ X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added an Ian onesie & a 'streamlined' hardsuit alternative.

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/Head/helmets.yml [hunk 1]
```diff
@@ -246,4 +246,55 @@
...
+  components:
+  - type: Sprite
+    sprite: Clothing/Head/Helmets/ancientvoidsuit.rsi
+  - type: Clothing
+    sprite: Clothing/Head/Helmets/ancientvoidsuit.rsi
+  - type: IngestionBlocker
+  - type: Tag
+    tags:
+    - HelmetEVA
+
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Clothing/Head/helmets.yml:286]: Revert

File: Resources/Prototypes/Entities/Clothing/Head/hardsuit-helmets.yml [hunk 2]
```diff
@@ -634,3 +634,26 @@
...
+    highPressureMultiplier: 0.08
+    lowPressureMultiplier: 1000
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.87
+        Slash: 0.87
+        Heat: 0.95
+        Radiation: 0.9
+        Caustic: 0.85
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Clothing/Head/hardsuit-helmets.yml:659]: This shouldn't be a hardsuit helmet.

Also why is the inhands colour different to when worn?

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Comment by deltanedas** [comment]
could add onesie as a rare hop locker spawn like backpack

File: Resources/Prototypes/Entities/Clothing/Head/hardsuit-helmets.yml (hunk 2 above)
Comment (QTNeen) [reply Resources/Prototypes/Entities/Clothing/Head/hardsuit-helmets.yml:659]: There's like 3 different versions of the original carp suit, one being a "hardsuit", where should I put this one?

No good with sprites, for which entity?

File: Resources/Prototypes/Entities/Clothing/Head/helmets.yml (hunk 1 above)
Comment (QTNeen) [reply Resources/Prototypes/Entities/Clothing/Head/helmets.yml]: I may have made a mistake in reverting this, I think I committed some stuff which shouldn't have been?

**Comment by deltanedas** [comment]
diffs completely fucked

**Comment by Emisse** [comment]
you have giga mapping changes in this that should be reverted

**Comment by QTNeen** [comment]
I require a re-do button I believe, @Emisse is it cool if I close this and re-pr it later?

**Comment by deltanedas** [comment]
still fucked, id just create new branch and add everything back manually

**Pull request closed without merging**